### DISCUSSION TITLE
Help for Fine Tuning download

### DISCUSSION THREAD

**Discussion by l5oukz89** [martinrotter/rssguard#1402] [Q&A]
let's say I have a feed that gathers all articles containing words "IT" and "crowd" but that's not enough filtered as i'm looking for exactely "IT Crowd", what should I write in the "post processing script" section to only keep articles where the occurrence is exactly "IT Crowd" ?

**Comment by l5oukz89** [comment]
@martinrotter 

when i fetch for example this feed (or any other feed containing more than one word like for artist name and family name)

https://www.reddit.com/search/.xml?q=title%3A("IT Crowd")+self%3Afalse+nsfw%3Ano&type=link&cId=c7562f6a-aac0-4f11-97f5-1950c3cee7a2&iId=5fe9374a-0b68-4fe6-aa07-25c5931eac42&sort=new&t=all.rss

Even with the quotes " " which are supposed to mean exact match, I still get articles containing "IT" and then after / before it a bunch of random words and then the word "Crowd" which wasn't the desired result as its specifically to find articles related to exactly "IT Crowd".

so i am guessing it's something related to regex which should be written in the post processing script of the feed settings or maybe modify the word between brackets with the coding language.

any idea on how to resolve that ?

**Comment by Ac314** [comment]
Try to play with article filters: https://rssguard.readthedocs.io/en/stable/features/filters.html

**Comment by razielfayad** [comment]
you can use an article filter

```javascript
const regex = /\b(IT Crowd)\b/i;

function filterMessage() {
  if (regex.test(msg.title)) {
    return MessageObject.Accept;
  }
  return MessageObject.Purge;
}
```
